doc/pages/pages.dox

```diff
@@ -11,8 +11,8 @@
 
 \subpage lipm_dcm_details
 
-<--! When incuding html files generated by lyx, replace the line <link rel="stylesheet" href="http://elyxer.nongnu.org/lyx.css" type="text/css" media="all"/>
- with <link rel="stylesheet" href="https://pinchito.es/elyxer/lyx.css" type="text/css" media="all"/> to avoid https errors -->
+<!-- When incuding html files generated by lyx, replace the line <link rel="stylesheet" href="http://elyxer.nongnu.org/lyx.css" type="text/css" media="all"/>
+  with <link rel="stylesheet" href="https://pinchito.es/elyxer/lyx.css" type="text/css" media="all"/> to avoid https errors -->
```

Comment:
We could add a sed rule to the CI so that it does it for us ;)

Reply:
Actually we should